wfe/web-front-end.go

```diff
@@ -769,6 +769,7 @@ func (wfe *WebFrontEndImpl) challenge(authz core.Authorization, response http.Re
 	switch request.Method {
 	case "GET":
 		challenge := authz.Challenges[challengeIndex]
+		challenge.ValidationRecord = nil
```

Comment:
[Because it isn't in the spec](https://github.com/letsencrypt/boulder/pull/505#discussion_r35828146) ([acme-spec#205](https://github.com/letsencrypt/acme-spec/issues/205)).